### PR TITLE
MM-11603 Fix select_team ui to have failed request scenario

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add error screen when getTeams request fail

#### Ticket Link
[MM-11603](https://mattermost.atlassian.net/browse/MM-11603)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes

#### Device Information
This PR was tested on: [Ios emulator]

#### Screenshots
aug-31-2018 12-54-45

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,6 +94,7 @@
     "jsdom-global": "3.0.2",
     "nyc": "11.7.1",
     "react-dom": "16.3.2",
+    "react-native-browser-polyfill": "0.1.2",
```
Comment (saturninoabril) [inline package.json]: Why do we need this?

File: app/screens/select_team/select_team.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// import 'react-native-browser-polyfill';
```
Comment (saturninoabril) [inline app/screens/select_team/select_team.test.js]: Maybe should remove this commented import.

File: package.json (hunk 1 above)
Comment (sudheerDev) [reply package.json]: Oops yea. it is not needed anymore. let me remove that. Thanks

**Comment by sudheerDev** [comment]
@saturninoabril Done. one thing i am adding do not merge label because i am waiting for assads comment on the UI.

**Comment by sudheerDev** [comment]
@enahum @saturninoabril Updated UI messages.

screen shot 2018-09-03 at 12 11 05 pm

Failure scenario of recent mentions has same UI messages as before
screen shot 2018-09-03 at 12 18 44 pm

**Comment by sudheerDev** [comment]
@enahum waiting for your approval on this?

**Pull request merged**